packages/dose3/dose3.5.0.1-2/files/0006-Grep-carefully-for-ocaml-system.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/configure.ac b/configure.ac
+index d73d724..3653943 100644
+--- a/configure.ac
++++ b/configure.ac
```

Comment:
Are you sure the patch works? I'm surprised to see only the `configure.ac` file patched and not the `configure` script that is actually used.

Reply:
Good point @kit-ty-kate , I only really tested this patch the dose3 repo, because I didn't want to set up my own opam repository. I suppose that `autoconf` is not part of the build (and in fact, it seems that `configure` is very much out of sync with `configure.ac`). I re-targeted the patch from `configure.ac` to `configure`.